cmd/nvkind/cluster_create.go

```diff
@@ -19,14 +19,25 @@ package main
 import (
 	"bufio"
 	"fmt"
+	"io"
+	"log"
 	"os"
+	"os/exec"
 	"time"
 
 	"github.com/urfave/cli/v2"
 
 	"github.com/NVIDIA/nvkind/pkg/nvkind"
 )
 
+const k8sRuntimeClassYamlDoc = `---
+apiVersion: node.k8s.io/v1
+handler: nvidia
+kind: RuntimeClass
+metadata:
+  name: nvidia
+`
+
```

Comment:
nit: remove line

go doesn't really have a double line break convention anywhere

Reply:
Ah, I see it now! It's good as is. I missed the backtick on the line by itself. Probably thought it was a speck on my screen :)